src/common/hooks/useSticky.tsx

```diff
@@ -1,40 +1,28 @@
-import { useEffect, useState } from 'react';
-import { useLocation } from 'react-router-dom';
-import { authRoutes } from '../../routes';
+import { useEffect } from 'react';
```

Comment:
The desired effect with `useSticky` was only different for the home page so I changed it so that all of the logic was specific to the home page. The useEffects that were being used will only be cleaned up if the component that they are watching is destroyed. This is why I made this change.